esrally/mechanic/provisioner.py

```diff
@@ -57,10 +54,12 @@ def docker(cfg, car, cluster_settings, ip, http_port, target_root, node_name):
 
 
 class NodeConfiguration:
-    def __init__(self, build_type, car_env, car_runtime_jdks, ip, node_name, node_root_path, binary_path, data_paths):
+    def __init__(self, build_type, car_env, car_runtime_jdks, car_provides_bundled_jdk, ip, node_name, node_root_path,
+                 binary_path, data_paths):
         self.build_type = build_type
         self.car_env = car_env
         self.car_runtime_jdks = car_runtime_jdks
+        self.car_provides_bundled_jdk = convert.to_bool(car_provides_bundled_jdk)
```

Comment:
Can we move the conversion (i.e. `convert.to_bool` to the call site(s)?) IMHO we should use the correct type `bool` already in the constructor.